### ISSUE TITLE
How to create catalog and add items with assets to it

### ISSUE THREAD

**Issue by habibutsu** [stac-utils/stac-fastapi#565]
I run stac-catalog with using pgstac, but it is not clear for me how to use it.

My expectation that i can take [pystac_client](https://pystac-client.readthedocs.io/en/stable/) and create Collection with some Items, but I don't see such methods for that.

I've found [this](https://github.com/stac-utils/pystac/blob/main/docs/tutorials/how-to-create-stac-catalogs.ipynb) notebook, but it shows how to create static-catalog in file

Could you provide example how to create items.

**Comment by habibutsu** [comment]
I found this variant. Is it right approach?

```python

import datetime as dt
from pystac import Collection,

collection = Collection(
    id="test",
    description="my first collection",
    extent=Extent(
        spatial=SpatialExtent(bboxes=[[-180, -90, 180, 90]]),
        temporal=TemporalExtent(intervals=[
            [dt.datetime(2016, 1, 1), dt.datetime(2022, 1, 1)]
        ])
    ))

r = requests.post(
    "http://127.0.0.1:8080/collections",
    json=collection.to_dict()
)
```

**Comment by habibutsu** [comment]
I've found [this](https://github.com/stac-utils/pystac-client/issues/241) issue in pystac-client. So, it's mean that there is only one way to create items - make requests manually.

**Comment by gadomski** [comment]
> make requests manually.

Correct. Check out https://github.com/stac-utils/stac-fastapi/blob/main/scripts/ingest_joplin.py for an example of a script to load data into a database using **stac-fastapi**.

We could have better documentation around this process, so I'm going to keep this issue open and change its title to make it a tracking issue for the docs.

**Comment by jonhealy1** [comment]
We have created a cli tool [here](https://github.com/stac-utils/stac-fastapi-elasticsearch-opensearch?tab=readme-ov-file#ingesting-sample-data-cli-tool)
